dask/bytes/core.py

```diff
@@ -19,8 +19,9 @@
 _open_text_files = dict()
 
 
-def read_bytes(path, delimiter=None, not_zero=False, blocksize=2**27,
-                     sample=True, compression=None, **kwargs):
+def read_bytes(urlpath, delimiter=None, not_zero=False, blocksize=2**27,
+                     sample=True, compression=None, storage_options=None,
+                     **kwargs):
```

Comment:
Why not just use kwargs for everything here

    kwargs = infer_storage_options(urlpath, kwargs)